test/inductor/test_pattern_matcher.py

```diff
@@ -454,6 +462,13 @@ def fn(a, b):
                 .contiguous()
                 .t(),
             ),
+        ]
+
+        for args in args_expect_mixed_mm:
+            check_uint4x2_mixed_mm(args, True)
+
+        # mixed mm is only enabled when casting from a lower-bitwidth dtype to a higher one
+        args_expect_no_mixed_mm = [
```

Comment:
@msaroufim Are there any cases where torchao has the weights in int64 and is trying to upcast it into fp32?

Reply:
No, at least not that i know of from https://github.com/pytorch/ao/blob/main/test/integration/test_integration.py

@HDCharles might have a use-case for that in mind though